nano/node/bootstrap/bootstrap_ascending.cpp

```diff
@@ -226,21 +294,8 @@ nano::account nano::bootstrap::bootstrap_ascending::account_sets::random ()
 nano::account nano::bootstrap::bootstrap_ascending::account_sets::next ()
 {
 	nano::account result;
-	if (!forwarding.empty ())
-	{
-		stats.inc (nano::stat::type::bootstrap_ascending_accounts, nano::stat::detail::next_forwarding);
-
-		auto iter = forwarding.begin ();
-		result = *iter;
-		forwarding.erase (iter);
-	}
-	else
-	{
-		stats.inc (nano::stat::type::bootstrap_ascending_accounts, nano::stat::detail::next_random);
-
-		result = random ();
-	}
-	backoff[result] += 1.0f;
+	stats.inc (nano::stat::type::bootstrap_ascending_accounts, nano::stat::detail::next_random);
```

Comment:
It is a bit surprising that next is equivalent to random unless random does not really mean random in which case maybe it should be renamed or at least documented.

Reply:
Removed.